tests/doctestUtils.cpp

```diff
@@ -0,0 +1,10 @@
+#include "doctestUtils.h"
+
+void fillWithRandomInts(std::list<int>* lst, int min, int max, int n) {
```

Comment:
Do you actually expect this function to be used by another outside of `ObjectPoolTests.cpp`? If not, it should be a helper function within that file, not in a separate file.